src/components/Login/Login.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useContext } from 'react'
+import { Link, useNavigate } from 'react-router-dom'
+import axios from 'axios'
+import { message } from 'antd'
+import { currentUserContext } from '../../helpers/context'
+import { RegisterLogin, Column, Span } from '../../common/CommonHTML'
+import Input from '../../common/Input'
+import Button from '../../common/Button'
+import getInfo from '../../helpers/getLoginOrRegisterInfo'
+
+export default function Login() {
+  const { setCurrentUser } = useContext(currentUserContext)
+  const navigate = useNavigate()
+  let loginInfo = {}
+  function inputInfo(e, field) {
+    const info = getInfo(loginInfo, e, field)
+    loginInfo = { ...info }
+  }
+
+  async function submitForm(e) {
+    e.preventDefault()
+    await axios.post('http://localhost:4000/login', loginInfo).catch(() => {
+      message.error('Please input email and password correctly!')
+    }).then((res) => {
+      window.localStorage.setItem('token', res.data.result)
+      if (window.localStorage.getItem('token')) {
+        setCurrentUser({ ...loginInfo })
+        navigate('/courses')
+      }
+    })
+  }

```

Comment:
We may split this function, there are a few steps/functions/parts:
1. send request
2. store the data received
3. navigation
4. error handling